packages/create/test/fixtures/ui5-deploy-config/ui5-deploy.yaml

```diff
@@ -0,0 +1,22 @@
+specVersion: '2.4'
+metadata:
+    name: 'fe_lrop_v4_odata_none'
+type: application
+builder:
+    resources:
+        excludes:
+            - /test/**
+            - /localService/**
+    customTasks:
+        - name: deploy-to-abap
+          afterTask: generateCachebusterInfo
+          configuration:
+              target:
+                  destination: ABC123
+                  url: https://abc.abap.stagingaws.hanavlab.ondemand.com
```

Comment:
Please make all mock/test urls end with `.example` as domain just to make sure that we don't by accident have a real url here